### PR TITLE
New version: yt-dlp.yt-dlp.nightly version 2024.01.31.232703

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137401)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-01-31 19:25:19.788 [CLI ] ShellExecute installer failed: 2 Installation failed with exit code -1978334972

(Automated response - build 694.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-01-31 19:25:19.788 [CLI ] ShellExecute installer failed: 2 Installation failed with exit code -1978334972

(Automated response - build 694.)

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/y/yt-dlp/yt-dlp/nightly/2024.01.31.232703/yt-dlp.yt-dlp.nightly.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created with komac v2.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: yt-dlp.yt-dlp.nightly
+PackageVersion: 2024.01.31.232703
+InstallerLocale: en-US
+InstallerType: exe
```
Comment (stephengillie) [inline manifests/y/yt-dlp/yt-dlp/nightly/2024.01.31.232703/yt-dlp.yt-dlp.nightly.installer.yaml:7]:
```suggestion
InstallerType: portable
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334972

(Automated response - build 694.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978334972

(Automated response - build 694.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**